ansible_community_steering_committee.rst

```diff
@@ -2,31 +2,28 @@
 Ansible Community Steering Committee
 ************************************
 
-The Steering Committee’s role is to provide continuity, guidance, and
-suggestions to the Ansible community, and to ensure delivery of the Ansible
-Community package. The Steering Committee should reflect the scope and
-breadth of the Ansible Community.
-
 Mission:
 ========
 
-The Steering Committee helps decide the technical direction of the
-Ansible project and is also responsible for avowal of new
-proposals (feature idea, new policy in collection world), new collection
-requests, empower contributors, maintainers & reviewers and other
-technical aspects regarding inclusion and packaging.
+The Steering Committee’s mission (hereinafter the Committee) is to provide continuity, guidance, and
+suggestions to the Ansible community, to ensure delivery and high quality of the Ansible package.
+
+The Committee helps decide the technical direction of the Ansible project and is also responsible for avowal of new
+proposals and policies in the community, package, and collections' world, new collection-inclusion requests,
+and other technical aspects regarding inclusion and packaging.
```

Comment:
`avowal` might not be the best term for an international audience, how about something like

```suggestion
The Committee helps decide the technical direction of the Ansible project. Its primary focus is to ensure that:
* proposals
* policies
* new collection inclusion requests
* (add other items here)
are adequately reviewed and progressed.
```

Reply:
I agree about `avowal` 100% - it confused me a bit when i saw it the first time:)
I've replaced it with `approval`. Is it OK?
I think the bullet point list would be an extra thing here because we have it a bit below describing things in detail. I would keep the `mission` paragraph as general as possible.